bookserver/routers/books.py

```diff
@@ -152,13 +164,13 @@ async def serve_page(
         # _`root_path`: The server is mounted in a different location depending on how it's run (directly from gunicorn/uvicorn or under the ``/ns`` prefix using nginx). Tell the JS what prefix to use for Ajax requests. See also `setting root_path <setting root_path>` and the `FastAPI docs <https://fastapi.tiangolo.com/advanced/behind-a-proxy/>`_. This is then used in the ``eBookConfig`` of :doc:`runestone/common/project_template/_templates/plugin_layouts/sphinx_bootstrap/layout.html`.
         new_server_prefix=request.scope.get("root_path"),
         # TODO
-        user_email="bonelake@mac.com",
+        user_email=user.email if logged_in else "",
```

Comment:
Should be `if user` instead, since `logged_in` is always true.

Reply:
got it! Thanks.